src/System.Runtime.WindowsRuntime/src/System/Windows/Size.cs

```diff
@@ -54,7 +53,7 @@ public double Width
             set
             {
                 if (value < 0)
-                    throw new ArgumentOutOfRangeException(SR.ArgumentOutOfRange_NeedNonNegNum, nameof(Width));
```

Comment:
I can do this, but why? The original parameter name was incorrect, typically we use "value" for the parameter name for an exception in the setter, right?